server/server.js

```diff
@@ -118,3 +118,7 @@ app.get('/server/status', (req, res) => {
   const outputJson = { status: 'True' };
   res.status(200).send(outputJson);
 });
+
+app.get('*', (req, res) => {
+  res.sendFile(path.join(__dirname, '..', '/client', '/page_404.html'));
```

Comment:
Don't need the slashes before client and page_404.html.

Reply:
oh right, fixing it.